spec/services/discovery_engine/sync/delete_spec.rb

```diff
@@ -56,41 +56,13 @@
     end
 
     it "does not set the remote version" do
-      expect(redis_client).not_to have_received(:set)
+      expect(version_cache).not_to have_received(:set_as_latest_synced_version)
     end
 
     it "logs that the document hasn't been deleted" do
       expect(logger).to have_received(:add).with(
         Logger::Severity::INFO,
-        "[DiscoveryEngine::Sync::Delete] Ignored as newer version (42) already synced content_id:some_content_id payload_version:1",
-      )
-    end
-  end
-
-  context "when there is no remote version yet" do
```

Comment:
Note: We no longer need to test this as the logic is now in `DocumentVersionCache` and tested there.